README.md

```diff
@@ -92,6 +92,7 @@ Supported methods:
 - [x] [PointRend (CVPR'2020)](configs/point_rend)
 - [x] [CGNet (TIP'2020)](configs/cgnet)
 - [x] [SETR (CVPR'2021)](configs/setr)
+- [x] [DPT (ArXiv' 2021)](configs/dpt)
```

Comment:
Please also add it to https://github.com/open-mmlab/mmsegmentation/blob/master/README_zh-CN.md.